playwright.config.ts

```diff
@@ -36,10 +36,10 @@ export default defineConfig({
     //     },
     //   },
     // },
-    {
-      name: 'webkit',
-      use: { ...devices['Desktop Safari'] },
-    },
+    // {
+    //   name: 'webkit',
+    //   use: { ...devices['Desktop Safari'] },
+    // },
```

Comment:
this is the root level playwright config. If we comment this, the e2e GHA won't run on safari. Can we keep this uncomment?